app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -398,10 +406,31 @@ class LuciBuildService {
     return buildBucketClient.getBuild(request);
   }
 
-  /// Get builder list whose config is pre-defined in LUCI.
+  /// Gets builder list whose config is pre-defined in LUCI.
+  ///
+  /// Returns cache if existing. Otherwise make the RPC call to fetch list.
   Future<Set<String>> getAvailableBuilderSet({
     String project = 'flutter',
     String bucket = 'prod',
+  }) async {
+    final Uint8List? cacheValue = await cache.getOrCreate(
+      subCacheName,
+      'builderlist',
+      createFn: () => _getAvailableBuilderSet(project: project, bucket: bucket),
```

Comment:
Please correct me if i am wrong but the builder set can be a fairly large list, are there size limits to entries in the cache?

Reply:
I see only a limit on the cache entries rather than a specific cache size.. For this question, I am not 100% sure. Though the list is big, the size is several kb I guess? I would expect redis handles this with no problem.. But needs to validate. I will monitor it closely tomorrow morning if this lands today.